libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/helpers.ts

```diff
@@ -376,7 +376,7 @@ export function getAxisRangeForAxes(chart: Highcharts.Chart): IAxisRangeForAxes
     return yAxis
         .map((axis: UnsafeInternals) => pick(axis, ["opposite", "min", "max"]))
         .map(({ opposite, min, max }: any) => ({ axis: opposite ? "second" : "first", min, max }))
-        .reduce((result: IAxisRangeForAxes, { axis, min, max }: any) => {
+        .reduce((result: Record<string, any>, { axis, min, max }: any) => {
```

Comment:
can we type it ? 

Record<string, {minAxisValue:number, maxAxisValue:number} >

Reply:
yes, will fix this